docs/question/html.md

```diff
@@ -1,13 +1,32 @@
-# Using Markdown in questions
+# `question.html`
 
-HTML and custom elements are great for flexibility and expressiveness. However, they're not great for working with large amounts of text, formatting text, and so on. [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) is a lightweight plaintext markup syntax that's ideal for authoring simple but rich text. To enable this, PrairieLearn adds a special `<markdown>` tag to questions. When a `<markdown>` block is encountered, its contents are converted to HTML. Here's an example `question.html` that utilizes this element:
+## Template file
+
+All `question.html` files are rendered with the [Mustache](https://mustache.github.io/mustache.5.html) template engine. Outside direct variable substitution, you can use Mustache to do things like conditionally render elements. For example, you can use the following syntax to conditionally render a piece of HTML:
+
+```html title="question.html"
+{{#params.show}}
+<p>This will only show up if params.show is true</p>
+{{/params.show}}
+```
+
+!!! tip
+
+    If you use triple-braces (e.g. `{{{params.html}}}`) then raw HTML is substituted (don't use this unless you know you need it).
+
+## Markdown in questions
+
+HTML and custom elements are great for flexibility and expressiveness. However, they're
+not great for working with large amounts of text, formatting text, and so on.
+[Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) is a lightweight
+plaintext markup syntax that's ideal for authoring simple but rich text. PrairieLearn recommends you author all the text of your questions in Markdown.
```

Comment:
Nit: conventionally we don't usually hard-wrap Markdown.

Reply:
8de961fa8bacdf280e06dd8f5e73d124836d5b15